tutorials/promscale-install.md

```diff
@@ -0,0 +1,146 @@
+## 3. Up and Running: Install Prometheus, Promscale and TimescaleDB [](up-and-running)
+
+We recommend four methods to setup and install Promscale:
+1. [TOBS - The Observability Stack for Kubernetes ][tobs-github] (recommended for kubernetes deployments)
+2. Docker (instructions detailed below) 
+3. [Helm][promscale-helm-chart]
+4. [Bare Metal][promscale-baremetal-docs]
+
+> :TIP: See the [Promscale github installation guide][promscale-github-installation] for more information on installation options.
+
+For demonstration purposes, we will use Docker to get up and running with Promscale.
+
+The easiest way to get started is by using Docker images. Make sure you have Docker installed on your local machine ([Docker installation instructions][docker]).
+
+The instructions below have 4 steps:
+1. [Install TimescaleDB](#install-timescaledb)
+2. [Install Promscale](#install-promscale)
+3. [Install node_exporter](#install-node-exporter)
+4. [Install Prometheus](#install-prometheus)
+
+>:WARNING: The instructions below are local testing purposes only and should not be used to set up a production environment.
+
+### Step 1: Install TimescaleDB [](install-timescaledb)
```

Comment:
All of the previous pages used decimal sections "1.1", "2.3", etc.  Please use this here to stay consistent (Or change the top-level headline numbering system)

Reply:
Good catch, fixed